src/pcap/packet.cr

```diff
@@ -55,6 +55,31 @@ module Pcap
       tcp_header.length
     end
 
+    # Packet flags available via capture loop
```

Comment:
I'm afraid that no needs of example code here.
Please put a comment here like this.

`### Convenient accessor methods for IpHeader and TcpHeader`